packages/synapse-react-client/src/components/FeaturedToolsList/FeaturedToolsList.tsx

```diff
@@ -34,13 +44,15 @@ export const FeaturedToolsList: React.FunctionComponent<
 > = ({
   entityId,
   toolDetailPageURL,
+  toolURLColumnName,
   idColumnName = 'id',
   nameColumnName = 'name',
   descriptionColumnName = 'description',
   typeColumnName = 'type',
-  dateColumnName = 'dateAdded',
+  dateColumnName,
+  filterClause,
 }) => {
-  const sql = `SELECT "${idColumnName}", "${nameColumnName}", "${descriptionColumnName}", "${typeColumnName}", "${dateColumnName}" FROM ${entityId} ORDER BY ${dateColumnName} DESC LIMIT 3`
```

Comment:
Just select all columns (limit is already set to 3 rows!), and make the filter/order customizable (since they differ in ELITE vs NF)

Reply:
would prefer to use the sort API in the QueryResultBundle instead of modifying the SQL